src/js/components/AppListComponent.jsx

```diff
@@ -394,6 +394,8 @@ var AppListComponent = React.createClass({
         state.fetchState !== States.STATE_LOADING
     });
 
+    var totalColSpan = 8;
```

Comment:
In my opinion variable names should never be abbreviated thus this should be named `totalColumnSpan`.